Bylaws.mediawiki

```diff
@@ -80,6 +80,33 @@
 
 # The organization reserves the right to limit membership based on the capacity of the space.
 
+== Equipment ==
+
+Hacklab operates under a general principle of "all members are equal owners of the space". This principle applies to the tools and materials inside of hacklab as well. Unless something has been labeled as property of a specific member, please consider the tool yours. But remember that it is not just yours, it belongs to everybody. Pursuant to this explanation, the following rules can be summarized as:
+
+# Don't use tools in hacklab in a way that will prevent other members from using them. In other words: don't hog or break stuff.
+# Don't hurt yourself.
+
+=== What to do before using equipment ===
+# Don’t use anything without signing the waiver
+# If it can dismember you (or poison you, or otherwise harm you significantly) please don’t use it if you don’t know how
+# If you don’t know how to use it, ask around the membership for help. 
+## Someone brought this equipment in, so someone (if they’re still a member) knows how to use it
+## There may be documentation for it on the public or internal wiki. Check the tool for a QR code. It may link you to a wiki page with instructions.
```

Comment:
Sorry for this https://github.com/hacklabto/hacklab-bylaws/pull/11/files/190f2d7f7624c1eb4538b56e5cbd9d6ccc4a7a8b#r1776347716 I was referring to this heading/section.

Reply:
Specifically, we [already mention the waiver](https://github.com/hacklabto/hacklab-bylaws/blob/7fb44162ac3b9b424134997d64e11c5c55a3147c/Bylaws.mediawiki?plain=1#L71). And the rest is just tips, which isn't what bylaws are for.

Reply:
Ah gotcha! Agreed, I removed this section in commit 4077a1f86546a570d84e7a38bdd63b302a524881